mkosi/__init__.py

```diff
@@ -138,6 +138,7 @@
     "/usr/bin/systemd-tty-ask-password-agent"
 ]
 
+DRACUT_EXTRA_MODULES = [] # type: List[str]
```

Comment:
```suggestion
DRACUT_EXTRA_MODULES: List[str] = [] 
```

Reply:
Looking at the previous version of this PR, you did `list[str]` instead of `List[str]`, we don't do that yet, since we're still lower-bounded to Python 3.7 (I need to push my 3.9 branch), so that's why that didn't work. `List[str]` as annotation works.

> If you want to support older dracut releases as well it should be fine but we dont support downstream playing russian release roulette with components. Using new dracut with old systemd etc.

Wanting and having to live with what's generally available are two different things. :)